src/main/scala/sbtdatabricks/DatabricksPlugin.scala

```diff
@@ -183,8 +189,63 @@ object DatabricksPlugin extends AutoPlugin {
     }
   }
 
+  private lazy val executeCommandImpl: Def.Initialize[Task[Seq[CommandStatus]]] = Def.taskDyn {
+    Def.task {
+      val client = dbcApiClient.value
+      val language = dbcExecutionLanguage.value
+      val onClusters = dbcClusters.value
+      val (allClusters, _) = dbcFetchClusters.value
+      val commandFile = dbcCommandFile.value
+      val commandStatuses = Seq.empty[CommandStatus]
+      for (clusterName <- onClusters) {
+        // Duplicating aspects of code in DatabricksHttp.foreachCluster
+        // - this is due to inability to pass values between DatabricksHttp function
+        // calls within the foreachCluster
```

Comment:
why not? You should be able to do something like:
```scala
client.foreachCluster(onClusters, allClusters) { confirmedCluster =>
 var contextCorrect = false
 val contextId = client.createContext(language, confirmedCluster)
 while (!contextCorrect) {
   val contextStatus = client.checkContext(contextId, confirmedCluster)
   if (contextStatus.status == DBCContextRunning.status) {
     contextCorrect = true
   } else if (contextStatus.status == DBCContextError.status) {
     client.destroyContext(contextId, confirmedCluster)
     contextCorrect = true
     Def.task(throw new RuntimeException(s"The Execution Context '${contextId.id} returned an error. Terminated context"))
   }
 }

 // Attempt to issue a command
 val commandId = client.executeCommand(language, confirmedCluster, contextId, commandFile)
 var commandComplete = false
 ...
}
```

Then you will also not have to worry about handling `ALL_CLUSTERS` :)